app/controllers/v1/appointments_controller.rb

```diff
@@ -1,21 +1,20 @@
 class V1::AppointmentsController < ApplicationController
-  def index
-    render json: Appointment.active(current_user.id)
-  end
+  before_action :authenticate_user!
 
-  def new
-    @appointment = Appointment.new
-    @days = WorkingDay.order(:date)
+  def index
+    render json: Appointment
+      .where(user: current_user)
+      .page(params[:page]).per(params[:per_page])
   end
 
   def create
     result = MakeAppointment.call(
       params: appointment_params
     )
     if result.success?
-      redirect_to root_path
+      render json: {}, status: :ok
```

Comment:
Я не помню, сработает ли, но попробуй просто
```suggestion
      render status: :ok
```

Reply:
нет, так не работает, к сожалению)